src/fev/metrics.py

```diff
@@ -266,7 +320,8 @@ def compute(
             quantile_levels=quantile_levels,
             target_column=target_column,
         )
-        return np.nanmean(ql) / np.nanmean(np.abs(np.array(test_data[target_column])))
+        scale = np.clip(np.abs(np.array(test_data[target_column])), self.epsilon, None)
+        return np.nanmean(ql) / np.nanmean(scale)
```

Comment:
Imo, we only want `epsilon` to kick in if the denominator is `< epsilon`, but current code clips all values before taking the mean.
```suggestion
        return np.nanmean(ql) / max(self.epsilon, np.nanmean(np.abs(np.array(test_data[target_column]))))
```